src/otx/recipe/classification/h_label_cls/efficientnet_b0_light.yaml

```diff
@@ -18,6 +18,12 @@ scheduler:
     patience: 1
     monitor: val/accuracy
 
+metric:
+  class_path: otx.algo.classification.metrics.hlabel_accuracy.HLabelAccuracy
```

Comment:
I will propose a package hierarchy once the execution is done, but a quick sneak peak. Ideally, tt would be easier to access via API if metrics are be located in `otx.metrics`

Reply:
> I will propose a package hierarchy once the execution is done, but a quick sneak peak. Ideally, it would be easier to access via API if metrics are be located in `otx.metrics`

I 100% agree with you. We need to clean the folder architecture

Reply:
Can we import `HLabelAccuracy` in __init__.py in metrics, and then use `otx.algo.classification.metrics.HLabelAccuracy` to reduce the complexity a bit?

Reply:
I heard that @samet-akcay is planning to the restructuring for the algo folder, it could be handeled at that phase.